jokes.md

```diff
@@ -4,5 +4,10 @@
 
 * Why do the French only have one egg for breakfast?
   * Because one egg is un oeuf
-* What does a thesaurus eat for breakfast?
+  
+* How do pirates know that they are pirates? 
+  * They think, therefore they ARRRR!!!!!
+  
+* * What does a thesaurus eat for breakfast?
```

Comment:
@psackley99 it looks like the formatting of this markdown got messed up. Please fix!